### ISSUE TITLE
dbx deploy using named parameters error

### ISSUE THREAD

**Issue by flori567** [databrickslabs/dbx#681]
## Expected Behavior

Named parameters of job task should be send as json array, not as list:

Example deployment.yaml:

        tasks:
          - task_key: "test"
            existing_cluster_id: "<cluster_id>"
            python_wheel_task:
              package_name: "databricks"
              entry_point: "test"
              parameters: []
              named_parameters:
                - "--test_parameter=test"

## Current Behavior

Json payload of the requests looks like this:

```
'python_wheel_task': {
  'package_name': 'databricks',
  'entry_point': 'test_entrypoint',
  'parameters': [],
  'named_parameters': ['--test_parameter=test'}
 }
```

and results in the following error:

HTTPError: 400 Client Error: Bad Request for url: <databricks_host>/api/2.0/jobs/reset
 Response from server:
 { 'error_code': 'MALFORMED_REQUEST',
  'message': "Could not parse request object: Expected 'START_OBJECT' not "
             "'VALUE_STRING'\n"
             ' at [Source: (ByteArrayInputStream); line: 1, column: 388]\n'
             ' at [Source: java.io.ByteArrayInputStream@13f5c213; line: 1, '
             'column: 388]'}

as the API expects it to look like this instead:

`"named_parameters": {"test_parameter": "test"}`

## Context

Creating a Databricks workflow using dbx deploy.

## Your Environment

Azure Databricks

* dbx version used: 0.8.7
* Databricks Runtime version: 11.3 LTS (includes Apache Spark 3.3.0, Scala 2.12)

**Comment by snovik75** [comment]
Confirm the same issue. Cannot use named_parameters

**Comment by mullerpeter** [comment]
Linked to this issue #589, which has been fixed by #593. 

There has been no release since last November, but if you install dbx 0.8.8 `pip install git+https://github.com/databrickslabs/dbx` named_parameters are working.

**Comment by renardeinside** [comment]
hey @mullerpeter , 
0.8.8 is incoming on Tuesday next week, stay tuned!

**Comment by renardeinside** [comment]
Closed in the 0.8.8
